CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.6.0 (2021-03-05)
```

Comment:
```suggestion
# 4.6.0 (2021-03-08)
```
We should probably update the changelog date since we didn't release this on Friday.